src/api/Metadata.js

```diff
@@ -719,6 +723,72 @@ class Metadata extends File {
             this.errorHandler(e);
         }
     }
+
+    /**
+     * API URL for metadata suggestions
+     *
+     * @return {string} url for metadata suggestions
+     */
+    getMetadataSuggestionsUrl(): string {
```

Comment:
Should all this logic move to its own API file similar to `MetadataQuery`?

Reply:
I didn't think it was worth it for one additional endpoint. The `MetadataQuery` also employs a caching strategy, so it made sense to encapsulate it in a separate class.